File: src/javaFiles/DesafioQuatro.java
package javaFiles;

import java.text.DecimalFormat;
import java.util.Scanner;

public class DesafioQuatro {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        System.out.printf("Insira a primeira nota: ");
        Double nota1 = scan.nextDouble();

        System.out.printf("Insira a segunda nota: ");
        Double nota2 = scan.nextDouble();

        System.out.printf("Insira a terceira nota: ");
        Double nota3 = scan.nextDouble();

        Double nota1comPeso = (2.0 * nota1);
        Double nota2comPeso = (3.0 * nota2);
        Double nota3comPeso = (5.0 * nota3);

        Double notaTotal = (nota1comPeso + nota2comPeso + nota3comPeso) / 10;

        System.out.print("A sua nota foi: " + notaTotal);

    }
}

File: src/jsFiles/desafioQuatro/desafioQuatro.js
var nota1 = prompt("Insira a primeira nota: ");
var nota2 = prompt("Insira a segunda nota: ");
var nota3 = prompt("Insira a terceira nota: ");

var nota1comPeso = (2.0 * nota1);
var nota2comPeso = (3.0 * nota2);
var nota3comPeso = (5.0 * nota3);

var notaTotal = (nota1comPeso + nota2comPeso + nota3comPeso) / 10;

alert(`A sua nota foi: ${notaTotal}`);